chapters/10_chapter_9.tex

```diff
@@ -0,0 +1,329 @@
+\chapter{Fault Attacks} \label{c9_ninthchapter:cha}
+
+% Swap the definition of \abs* and \norm*, so that \abs
+% and \norm resizes the size of the brackets, and the 
+% starred version does not.
+\makeatletter
+\let\oldabs\abs
+\def\abs{\@ifstar{\oldabs}{\oldabs*}}
+%
+\let\oldnorm\norm
+\def\norm{\@ifstar{\oldnorm}{\oldnorm*}}
+\makeatother
+
+\title{
+    {\large 372-2-5421} \\
+    {\large \textsc{Attacks on Implementations of Secure Systems}} \\
+    \vspace{20pt}
+    \textbf{\textsc{Scribe notes} -- Fault Attacks}
+}
+\author{
+    \parbox{\linewidth}{\centering
+    Dorel Yaffe, Dan Arad, Iliya Fayans \\
+    \vspace{7pt}\texttt{\small{\{yaffed,arda,fayansi\}@post.bgu.ac.il}}
+    }
+}
+\date{\footnotesize{\textbf{2019-05-27}}}
+
+\maketitle
+
+\begin{centering}
+    \section*{Topics}
+        \begin{enumerate}
+            \begin{centering}
+            	\item Introduction to Fault Attacks
+            	\item Fault Attack on RSA-CRT
+            	\item DRAM and Rowhammer
+            	\item Flip-Feng-Shui: Rowhammer attack on RSA \\
+            \end{centering}
+        \end{enumerate}
+\end{centering}
+\newpage
+\section*{Introduction to Fault Attacks}
+Up until now in the course we learned about \emph{passive} attacks -- i.e. attacks which measure \emph{leakage} such as timing information and power traces. The advantage of these attacks is that they allow an attacker to acquire information in the process of an ongoing computation e.g. an AES key \emph{before} it was fully mixed with the input -- this fact can help the attacker extract secrets.
+
+In fault attacks we will become \emph{active} in the sense that we will give the device-under-test (DUT) additional inputs such as heat or radiation.
+
+One problem with Fault attacks: they use the strongest attack model, meaning -- we assume most power on the attacker's part.
+
+\section{Active Attacks}
+\paragraph{Definition:} 
+\begin{quote}
+    \textit{"A fault attack is an active attack that allows extraction of secret information from cryptographic devices by breaking those devices."}
+\end{quote}
+
+In fault attacks we are being \emph{active} -- we give additional inputs beside the main input such as:
+\begin{enumerate}
+	\item Fuzzing (garbage or bad input)
+	\item Radiation
+	\item Heat
+	\item Vibration
+	\item Power spikes etc.
+\end{enumerate}
+
+This way, we receive other (usually erronous) outputs which might give us additional information about the computation and/or the secret. This process is described in figure \ref{fig:fault_attacks_schematic}.
```

Comment:
Please use cleveref \cref or \Cref for this.

Reply:
Changed to \cref.
I don't see the corresponding \usepackage declaration in main though